tests/playgrounds/screenshot.js

```diff
@@ -76,9 +75,12 @@ function workspaceToSvg_(workspace, callback, customCss) {
   svg.setAttribute("style", 'background-color: transparent');
 
   const css = [].slice.call(document.head.querySelectorAll('style'))
-      .filter(function(el) { return /\.blocklySvg/.test(el.innerText) ||
-        (el.id.indexOf('blockly-') === 0); }).map(function(el) {
-        return el.innerText; }).join('\n');
+      .filter(function(el) {
+        return /\.blocklySvg/.test(el.innerText) ||
+        (el.id.indexOf('blockly-') === 0);
+      }).map(function(el) {
+        return el.innerText;
+      }).join('\n');
```

Comment:
Perhaps use arrow functions?:

```suggestion
      .filter((el) => return /\.blocklySvg/.test(el.innerText) || (el.id.indexOf('blockly-') === 0))
      .map((el) => return el.innerText)
      .join('\n');
```

Reply:
Done, although note that the `return` is not required (and my editor seems to think that if I want `return` I should also have braces).

Reply:
Oh yeah.  Sorry: it turns out I'm just an ES6 poseur.